Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "manta-sdk"
+edition = "2021"
+version = "1.2.1"
```

Comment:
Where did this version number come from? Not sure this will work if we need to match the existing binaries.

Reply:
Comes from Kevin's previous PR: https://github.com/Manta-Network/sdk/commit/e248057d53c38661ae44e6f7c7e77212019b567c#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R4

In this PR, I only revert my last PR on removing the data library and did not add anything special.